File: IMSBackend/IMSWebAPI.Models/Product.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace IMSWebAPI.Models
{
    public class Product
    {
        [Key]
        public int ProductID { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public int Quantity { get; set; }
        public decimal PurchasePrice { get; set; }
        public decimal SalePrice { get; set; }
    }

}

File: IMSFrontend/src/app/modals/product.ts
export class Product {
    productID: number;
    sku: string;
    name: string;
    description: string;
    quantity: number;
    purchasePrice: number;
    salePrice: number;
    currentStock: number;
  
    constructor(
      productID: number,
      sku: string,
      name: string,
      description: string,
      quantity: number,
      purchasePrice: number,
      salePrice: number,
      currentStock: number
    ) {
      this.productID = productID;
      this.sku = sku;
      this.name = name;
      this.description = description;
      this.quantity = quantity;
      this.purchasePrice = purchasePrice;
      this.salePrice = salePrice;
      this.currentStock = currentStock;
    }
  }
  